index.html

```diff
@@ -1615,6 +1618,211 @@ <h3>
         <a>related applications</a> instead.
       </p>
     </section>
+    <section>
+      <h3>
+        `scope_extensions` member
+      </h3>
+      <p>
+        The [=manifest's=] <code><dfn data-export="" data-dfn-for="manifest">
+          scope_extensions</dfn></code> member represents a list of an
+          application's desired [=scope extensions=].
+      </p>
+      <p>
+        A <dfn>scope extension</dfn> extends the [=manifest/navigation scope of
+        a manifest=] by describing additional [=URLs=] that should be treated as
+        being [=within extended scope=].
+      </p>
+      <p>
+        The user agent MUST [=process the scope_extensions member=] and 
+        [=validate scope extensions=] before it can [=apply extended scope=] 
+        to allow additional [=URLs=] to be [=within extended scope=].
+      </p>
+      <p>
+        A [=URL=] |target:URL| is <dfn>within extended scope</dfn> of a
+        |manifest:processed manifest| if the |target| is [=manifest/within
+        scope=] of the manifest or [=matches a validated scope extension=]. 
+      </p>
+      <p>
+        A [=URL=] |target:URL| <dfn>matches a validated scope extension</dfn> if
+        the following algorithm returns `true`, given |target| and [=ordered map=]
+        |validated_scope_extension:ordered map|:
+      </p>
+      <ol class="algorithm">
+        <li>Let [=URL=] |resolved_scope:URL| be
+          |validated_scope_extensions:ordered map|["scope"]. </li>
+        <li>If |target| is [=URL/within scope=] of |resolved_scope|, return
+          `true`.</li>
+        <li>Return `false`.</li>
+      </ol>
+      <aside class="note">
+        The [=manifest's=] `scope` member only allows URLs to be added to scope
+        from a single [=origin=]. The `scope_extensions` member allows URLs to
+        be added from multiple [=origins=].
```

Comment:
'added' might  be a bit weird here

```suggestion
        The [=manifest's=] `scope` member only allows URLs to be considered for scope
        from a single [=origin=]. The `scope_extensions` member allows URLs to
        be considered from multiple [=origins=].
```